TM1py/Services/CellService.py

```diff
@@ -406,6 +456,7 @@ def write(self, cube_name: str, cellset_as_dict: Dict, dimensions: Iterable[str]
             deactivate_transaction_log=deactivate_transaction_log,
             reactivate_transaction_log=reactivate_transaction_log,
             sandbox_name=sandbox_name,
+            use_changeset=True,
```

Comment:
this should say `use_changeset=use_changeset` I think

Reply:
fixed